[CF1] microsoft entra ID IdP guide revision (#21792)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/identity/idp-integration/entra-id.mdx

To retrieve those values:

1.  Log in to the [Microsoft Entra admin center](https://entra.microsoft.com/).

2.  Go to **Applications** > **Enterprise applications**.

3.  Select **New application**, then select **Create your own application**.

4.  Name your application.

> 5.  Select **Register an application to integrate with Microsoft Entra ID (App you're developing)**. If offered, do not select any of the gallery applications. Select **Create**.

Reviewer: ```suggestion
5.  Select **Register an application to integrate with Microsoft Entra ID (app you are developing)**. If offered, do not select any of the gallery applications. Select **Create**.
```

> 7.  Under **Redirect URI**, select the _Web_ platform and enter the following URL.

Reviewer: ```suggestion
6.  Under **Redirect URI**, select the _Web_ platform and enter the following URL.
```

    ```txt
    https://<your-team-name>.cloudflareaccess.com/cdn-cgi/access/callback
    ```

    In [Zero Trust](https://one.dash.cloudflare.com), go to **Settings** > **Custom Pages** to find your team name.

    ![Registering an application in Azure](~/assets/images/cloudflare-one/identity/azure/name-app.png)

> 8.  Select **Register**.

Reviewer: ```suggestion
7.  Select **Register**.
```

> 9.  Next, return to Microsoft Entra ID and go to **Applications** > **App registrations**.

Reviewer: ```suggestion
8.  Next, return to Microsoft Entra ID and go to **Applications** > **App registrations**.
```

> 10. Select **All applications** and select the app you just created. Copy the **Application (client) ID** and **Directory (tenant) ID**. You will need these values when [adding Entra ID as an identity provider in step 3](/cloudflare-one/identity/idp-integration/entra-id/#3-add-entra-id-as-an-identity-provider).

Reviewer: ```suggestion
9. Select **All applications** and select the app you just created. Copy the **Application (client) ID** and **Directory (tenant) ID**. You will need these values when [adding Entra ID as an identity provider in step 3](/cloudflare-one/identity/idp-integration/entra-id/#3-add-entra-id-as-an-identity-provider).
```

    When the client secret expires, users will be unable to log in through Access. Take note of your expiry date to prevent login errors and renew your client secret when necessary.
    :::

> 13. After the client secret is created, copy its **Value** field. Store the client secret in a safe place, as it can only be viewed immediately after creation. You will need this client secret value when [adding Entra ID as an identity provider in step 3](/cloudflare-one/identity/idp-integration/entra-id/#3-add-entra-id-as-an-identity-provider).

Reviewer: I can't seem to edit all of the numbering - but the numbered list needs to shift down

        }'
        ```

</TabItem> <TabItem label="Terraform">

:::note[Provider versions]

> The following example requires Cloudflare provider version `>=4.40.0`.

Reviewer: ```suggestion
The following example requires Cloudflare provider version `4.40.0` or later.
```
Nit
